pubspec.yaml

```diff
@@ -10,15 +10,33 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  args: '>=0.13.7 <2.0.0'
+  args: ^1.5.0
+  checked_yaml: ^1.0.0
+  glob: ^1.2.0
+  json_annotation: ^3.0.0
   logging: ^0.11.3+1
   path: ^1.4.2
   pub_semver: ^1.4.1
   yaml: ^2.1.13
 
 dev_dependencies:
+  build_runner: ^1.7.3
+  json_serializable: ^3.2.5
   test: ^1.9.4
   test_descriptor: ^1.2.0
+  workiva_analysis_options: ^1.0.0
 
 executables:
   dependency_validator:
+
+dependency_validator:
+  exclude:
+    - 'test_fixtures/**'
+  ignore:
+    - json_serializable
```

Comment:
note to self: we should look up dependencies in the pub-cache to see if they provide a `build.yaml` that defines a builder - if that's the case, it's likely that the dependency is needed for its builder and shouldn't need to be ignored.